config/initializers/devise.rb

```diff
@@ -280,4 +281,6 @@
   # When using OmniAuth, Devise cannot automatically set OmniAuth path,
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
+  config.secret_key = '0a37102ea08449d2150786350d49ccac0a135ddd'\
```

Comment:
don't pass your sensitive data in repository, and even more TO PRODUCTION

with this key, accounts on your website can be easily stolen